src/Microsoft.DotNet.Interactive.Browser/Microsoft.DotNet.Interactive.Browser.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why not `netstandard2.1`?

Reply:
I'm not sure it's useful. The functionality here is never going to work in WASM, which is the primary reason we  support `netstandard2.1` in some packages.